### ISSUE TITLE
[TD] n-Hardened conduit into 1 redstone energy conduit mess the rf/t amount, pipes get full

### ISSUE THREAD

**Issue by Maxiride** [CoFH/Feedback#1080]
As I initially explained [here](https://github.com/BluSunrize/ImmersiveEngineering/issues/968) I have several hardened conduit merging into one redstone conduit.

The bug I'm experiencing is that if I take each group of hardened conduit one by one, the RF/t transmitted is right.
However when all the groups of hardened conduit are connected the rf/t transmitted doesn't match the sum of the previous groups taken one by one and WAILA shows that the redstone energy conduit is almost empty, when instead some of the hardened conduits gets the buffer full.

I honestly don't know how to better test it, after some messing around I just removed and replaced all the pipes and the sum now does match as shown in the new [screenshots](https://imgur.com/a/VluEI) even if the overall maximum rf/t didn't change.

**Comment by KingLemming** [comment]
This sounds like you don't understand how fluxducts (not conduit) work.

RF/t is per connection. So you're limiting them HEAVILY with this setup as your overall transfer is limited to the single connection between the hardened/redstone.

Run redstone everywhere if you want to do something like this.

**Comment by Maxiride** [comment]
I thought to have understood how the per connection thing worked, in fact there is a [screenshot](https://i.imgur.com/mN4WiqL.png) which shows that the hardened fluxducts are connected in group of 4 at the thermopiles each one producing 62 rf/t hence each group of 4 outputs 248 rf/t (below the 800 rf/t of the hardened fluxducts).

On the vertical redstone flux duct where all the hardened join there are 5 connection each one providing 248 rf/t for a total of 1250 rf/t. 

So for each fluxduct the rf/t limit is respected in that design, however the rf/t output doesn't match the sum of the 4 groups.

I'll switch to all redstone fluxducts but I don't understand why this is happening given that in the design shown in this comment the rf/t per connection is respected.

**Comment by KingLemming** [comment]
Fair enough then. Honestly I just sort of assumed that since thermopiles are IE, their output was way ridiculously overpowered, so 800 RF/t wouldn't cut it.

**Comment by Maxiride** [comment]
Thanks for the suggestions and help ;) I'll switch to all redstone fluxducts.

**Comment by codewarrior0** [comment]
Some weeks ago, I ended up building an identical setup with thermopiles and mixed fluxducts. I saw the exact same thing, where the hardened fluxducts would stay full while the redstone ones would be empty or nearly empty, and the total power output (measured using an EnderIO capacitor) was much less than expected. I got the feeling that power was sloshing between the two networks. I ended up replacing them all with redstone fluxducts so there was only one network.  After reading that RF is mostly push-based, I'm wondering if the redstone network really was pushing the power right back into the hardened one.
